docs/docs/examples/workflow/citation_query_engine.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        "{context_str}"
</pre><p>is the context actually numbered here? Ravi/Logan </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
oh i missed in create_citation_nodes